telebot/__init__.py

```diff
@@ -1172,6 +1184,9 @@ def polling(self, non_stop: Optional[bool]=False, skip_pending: Optional[bool]=F
         if restart_on_change:
             self._setup_change_detector(path_to_watch)
 
+        if not self._user:
```

Comment:
We discussed that get_me should be optional.

Reply:
Yep, my bad, for sync there's no need for this as self.user is a property that already does this

Reply:
oh hm, the use case might be where validate_token is false, will fix